### DISCUSSION TITLE
Theme changing

### DISCUSSION THREAD

**Discussion by Jaklik** [leezng/vue-json-pretty#268] [Q&A] [answered]
Hi, I'm trying to change the theme according to the documentation, but it doesn't seem to work completely. When I looked in the generated files, the props theme is not there at all. I'm using Nuxt 3. Maybe it's just my stupidity, but the other props from the doc work. I have a color inversion based switcher, the text changes fine, but the line highlighting stays the same and the data is not readable on a dark background. I am using v. 2.3.0. 

There are props generated in my code by this package
![Screenshot from 2024-02-25 13-13-42](https://github.com/leezng/vue-json-pretty/assets/57942435/77ee940e-7868-4026-8c42-1ff202ceae06)

Thank you!

**Comment by leezng** [comment, marked as answer]
Waiting for release
